db/seeds.rb

```diff
@@ -6,31 +6,68 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+# puts "seeds.rb"
+
 # the '99' is to make sure we leave space for the migrated divisions
-Division.find_or_create_by({id:99, internal_name: Division.root_internal_name, name:'Root Division'})
+Division.find_or_create_by(id: 99, internal_name: Division.root_internal_name, name:'Root Division')
 # note, divisions created within the new system will start at 101
 Division.connection.execute("select setval('divisions_id_seq', greatest((select max(id) from divisions), 100))")
 
-# note, these hardcoded id's are needed to match the migrated data
-Language.find_or_create_by({id:1,name:'English',code:'EN'})
-Language.find_or_create_by({id:2,name:'Español',code:'ES'})
-Language.find_or_create_by({id:3,name:'Français',code:'FR'})
+
+Language.find_or_create_by(id: 1, name: 'English', code: 'EN')
+Language.find_or_create_by(id: 2, name: 'Español', code: 'ES')
+Language.find_or_create_by(id: 3, name: 'Français', code: 'FR')
```

Comment:
I'm wondering out loud (as it were) if there's an easy way to avoid having to specify IDs. Perhaps using something like an old_db_id column and using that to match associations in the migration code. On the other hand, I'm also wondering if it's a big enough deal to bother setting that up. It definitely smells bad to specify IDs in the code, even in seeds I think. But I'm having trouble thinking of an actual practical problem that might arise from it.